main.go

```diff
@@ -83,9 +84,22 @@ func main() {
 		Desc:   "K8s label used to enable/disable automatic deployments.",
 		EnvVar: "LABEL",
 	})
+	logLevel := app.String(cli.StringOpt{
+		Name:   "log",
+		Value:  "warn",
+		Desc:   "Log level [trace|debug|info|warn|error] case insensitive",
+		EnvVar: "LOG_LEVEL",
+	})
+
+	log.Logger = hclog.New(&hclog.LoggerOptions{
```

Comment:
Good point!

I'm struggling to figure out a name, is it non-idiomatic to overload default so: `func Init(logLevel string)` or is it okay?

Reply:
I would call it something like `InitLogger` to avoid using the default function, as its use is to be called on package initialisation  (but here we call it after), but I am not sure about the best practice on that. I will ask around to see what others people think